code/modules/transport/tram/tram_machinery.dm

```diff
@@ -58,17 +58,17 @@
 				if(BROKEN_BEYOND_REPAIR)
 					say("The tram has suffered a catastrophic failure. Please seek alternate modes of travel.")
 				if(NOT_IN_SERVICE) //tram has no power or other fault, but it's not broken forever
-					say("The tram is not in service. Please contact the nearest engineer.")
+					say("The tram is not in service due to loss of power or system problems. Please contact the nearest engineer to check power and controller.")
 				if(INVALID_PLATFORM) //engineer needs to fix button
 					say("Button configuration error. Please contact the nearest engineer.")
 				if(TRANSPORT_IN_USE)
 					say("The tram is tramversing the station, please wait.")
 				if(INTERNAL_ERROR)
-					say("Tram controller error. Please contact the nearest engineer.")
+					say("Tram controller error. Please contact the nearest engineer or crew member with telecoms access to reset the controller.")
```

Comment:
Same comment on this and a following line about `telecomms`

Reply:
Sure